PhysicalTherapy/PhyscialTherapy/Client/src/app/login/login.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+
+@Component({
+    selector: 'app-login',
+    templateUrl: './login.component.html',
+    styleUrls: ['./login.component.scss']
+})
+export class Login { 
+    constructor(private router: Router, private http: HttpClient){ }
+
+    // TODO: Add routing to patient homepage
+    loginClick(username, password, error) {
+        error.innerText = '';
+        this.http.get(`http://localhost:64496/api/Credentials/${username.value}`)
```

Comment:
Instead of hardcoding the localhost part of the url, should we set a global variable somewhere and then concatenate whatever relevant parts remain ( hostURL + "/Credentials/${username.value}"?

Reply:
I was actually wondering about that when I put it in. I think you are correct. We'll need to figure out where the best spot for those variables are and keep as few things global as we can. But I imagine when we actually deploy it we will definitely need things like that to be changed.

Reply:
So in this case maybe: export const apiEndpoint = "http://localhost:64496/api";

Then we can use that any time we need to access one of our api endpoints.

Reply:
That sounds good. Do you want to add it to these changes, or should we do it later?

Reply:
Done.